src/frontend/src/eth/components/transactions/EthTransactionModal.svelte

```diff
@@ -124,14 +125,16 @@
 
 		<Value ref="amount">
 			<svelte:fragment slot="label">{$i18n.core.text.amount}</svelte:fragment>
-			<output>
-				{formatToken({
-					value,
-					unitName: $tokenWithFallback.decimals,
-					displayDecimals: $tokenWithFallback.decimals
-				})}
-				{$tokenWithFallback.symbol}
-			</output>
+			{#if nonNullish(token)}
```

Comment:
Then you have the label "Amount:" but no amount? Shoudn't the all block be hidden if no token is provided?

Reply:
ideally the token MUST be provided, it is a safeguard only because for now we still use the `token` store

i replicated what we were doing in `IcTransactionModal`, but I think the idea is that if for some reason there is a lag in loading the prop token, at least we already have the `Amount:` label and then we load the amount?

Not really sure UX-wise what is better, WDYT?

Reply:
Personally I would hide the all block if not provided. If you want to keep the amount, then you should probably add a `&ZeroWidthSpace;` for the empty value otherwise there might be a lack of space in the UI?

Reply:
let's hide the entire block, makes more sense